[2022-02-17 10:29:38,645] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_login_organization
[2022-02-17 10:29:40,758] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_add_orgcom
[2022-02-17 10:29:43,400] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_edit_orgcom
[2022-02-17 10:29:45,160] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_delete_orgcom
[2022-02-17 10:29:46,386] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_refresh_orgcom
[2022-02-17 10:29:46,495] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_org_user_edit
[2022-02-17 10:29:53,113] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_org_user_delete
[2022-02-17 10:29:53,250] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_org_refresh
[2022-02-17 10:32:32,753] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_login_organization
[2022-02-17 10:32:35,160] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_add_orgcom
[2022-02-17 10:32:38,968] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_edit_orgcom
[2022-02-17 10:32:40,680] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_delete_orgcom
[2022-02-17 10:32:41,928] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_refresh_orgcom
[2022-02-17 10:32:42,875] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_org_user_edit
[2022-02-17 10:32:49,218] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_org_user_delete
[2022-02-17 10:32:49,308] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_org_refresh
[2022-02-17 10:35:43,353] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_login_organization
[2022-02-17 10:35:45,960] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_add_orgcom
[2022-02-17 10:35:48,988] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_edit_orgcom
[2022-02-17 10:35:50,916] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_delete_orgcom
[2022-02-17 10:35:52,171] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_refresh_orgcom
[2022-02-17 10:35:52,309] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_org_user_edit
[2022-02-17 10:35:59,175] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_org_user_delete
[2022-02-17 10:35:59,290] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_org_refresh
[2022-02-17 10:41:50,711] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_add_orgcom_name_abnormal
[2022-02-17 10:41:55,549] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_add_orgcom_name_abnormal
[2022-02-17 10:42:05,572] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_add_orgcom_name_abnormal
[2022-02-17 10:42:43,202] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_add_orgcom_code_abnormal
[2022-02-17 10:43:25,054] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_add_orgcom_code_abnormal
[2022-02-17 10:44:07,256] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_add_orgcom_code_abnormal
[2022-02-17 10:44:49,890] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_add_orgcom
[2022-02-17 10:45:32,490] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_edit_orgcom_name_abnormal
[2022-02-17 10:46:15,095] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_edit_orgcom_name_abnormal
[2022-02-17 10:46:57,683] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_edit_orgcom_name_abnormal
[2022-02-17 10:47:40,485] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_org_username_add
[2022-02-17 10:48:01,153] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_org_username_add
[2022-02-17 10:48:21,912] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_org_username_add
[2022-02-17 10:48:42,402] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_org_code_add
[2022-02-17 10:49:02,942] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_org_code_add
[2022-02-17 10:49:23,570] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_org_code_add
[2022-02-17 10:49:44,213] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_org_code_add
[2022-02-17 10:50:04,796] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_org_code_add
[2022-02-17 10:50:25,504] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_org_tel_add
[2022-02-17 10:50:46,196] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_org_tel_add
[2022-02-17 10:51:06,882] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_org_tel_add
[2022-02-17 10:51:27,534] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_org_tel_add
[2022-02-17 10:51:48,227] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_org_username_edit
[2022-02-17 10:52:09,911] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_org_username_edit
[2022-02-17 10:52:31,690] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_org_username_edit
[2022-02-17 10:52:53,300] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_org_tel_edit
[2022-02-17 11:07:41,337] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_add_orgcom_name_abnormal
[2022-02-17 11:07:44,165] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_add_orgcom_name_abnormal
[2022-02-17 11:26:32,204] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_add_orgcom_name_abnormal
[2022-02-17 11:49:00,865] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_add_orgcom_name_abnormal_001
[2022-02-17 11:49:02,674] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_add_orgcom_name_abnormal_002
[2022-02-17 11:49:03,563] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_add_orgcom_name_abnormal_003
[2022-02-17 11:49:04,452] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_add_orgcom_code_abnormal_001
[2022-02-17 11:49:06,176] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_add_orgcom_code_abnormal_002
[2022-02-17 11:49:17,772] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_add_orgcom_code_abnormal_003
[2022-02-17 11:49:35,805] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_add_orgcom
[2022-02-17 11:49:41,632] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_edit_orgcom_name_abnormal_001
[2022-02-17 11:49:42,390] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_edit_orgcom_name_abnormal_002
[2022-02-17 11:49:48,469] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_edit_orgcom_name_abnormal_003
[2022-02-17 13:37:49,047] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_add_orgcom_name_abnormal_001
[2022-02-17 13:37:53,871] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_add_orgcom_name_abnormal_002
[2022-02-17 13:37:56,174] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_add_orgcom_name_abnormal_003
[2022-02-17 13:38:18,998] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_add_orgcom_name_abnormal_001
[2022-02-17 13:38:20,581] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_add_orgcom_name_abnormal_002
[2022-02-17 13:38:23,250] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_add_orgcom_name_abnormal_003
[2022-02-17 13:38:24,118] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_add_orgcom_code_abnormal_001
[2022-02-17 13:38:24,980] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_add_orgcom_code_abnormal_002
[2022-02-17 13:38:25,896] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_add_orgcom_code_abnormal_003
[2022-02-17 13:38:26,782] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_add_orgcom
[2022-02-17 13:38:28,612] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_edit_orgcom_name_abnormal_001
[2022-02-17 13:38:29,319] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_edit_orgcom_name_abnormal_002
[2022-02-17 13:38:44,788] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_edit_orgcom_name_abnormal_003
[2022-02-17 13:41:31,076] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_add_orgcom_name_abnormal_001
[2022-02-17 13:41:32,807] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_add_orgcom_name_abnormal_002
[2022-02-17 13:41:34,168] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_add_orgcom_name_abnormal_003
[2022-02-17 13:41:35,053] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_add_orgcom_code_abnormal_001
[2022-02-17 13:41:35,903] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_add_orgcom_code_abnormal_002
[2022-02-17 13:41:36,851] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_add_orgcom_code_abnormal_003
[2022-02-17 13:41:37,765] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_add_orgcom
[2022-02-17 13:41:39,632] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_edit_orgcom_name_abnormal_001
[2022-02-17 13:41:51,743] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_edit_orgcom_name_abnormal_002
[2022-02-17 13:41:52,782] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_edit_orgcom_name_abnormal_003
[2022-02-17 13:43:46,910] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_add_orgcom_name_abnormal_001
[2022-02-17 13:43:48,749] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_add_orgcom_name_abnormal_002
[2022-02-17 13:43:51,378] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_add_orgcom_name_abnormal_003
[2022-02-17 13:43:52,236] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_add_orgcom_code_abnormal_001
[2022-02-17 13:43:53,089] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_add_orgcom_code_abnormal_002
[2022-02-17 13:43:54,000] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_add_orgcom_code_abnormal_003
[2022-02-17 13:43:54,931] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_add_orgcom
[2022-02-17 13:43:56,775] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_edit_orgcom_name_abnormal_001
[2022-02-17 13:43:57,477] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_edit_orgcom_name_abnormal_002
[2022-02-17 13:43:59,418] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_edit_orgcom_name_abnormal_003
[2022-02-17 13:47:08,034] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_org_username_add_abnormal_001
[2022-02-17 13:47:12,713] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_org_username_add_abnormal_002
[2022-02-17 13:58:33,269] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_add_orgcom_name_abnormal_001
[2022-02-17 13:58:37,924] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_add_orgcom_name_abnormal_002
[2022-02-17 14:07:34,536] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_add_orgcom_name_abnormal_001
[2022-02-17 14:07:36,431] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_add_orgcom_name_abnormal_002
[2022-02-17 14:07:37,314] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_add_orgcom_name_abnormal_003
[2022-02-17 14:07:38,190] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_add_orgcom_code_abnormal_001
[2022-02-17 14:07:39,024] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_add_orgcom_code_abnormal_002
[2022-02-17 14:07:39,935] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_add_orgcom_code_abnormal_003
[2022-02-17 14:07:40,858] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_add_orgcom
[2022-02-17 14:07:42,695] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_edit_orgcom_name_abnormal_001
[2022-02-17 14:07:43,403] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_edit_orgcom_name_abnormal_002
[2022-02-17 14:07:44,420] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_edit_orgcom_name_abnormal_003
[2022-02-17 14:07:45,485] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_org_username_add_abnormal_001
[2022-02-17 14:07:48,672] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_org_username_add_abnormal_002
[2022-02-17 14:27:37,067] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_add_orgcom_name_abnormal_001
[2022-02-17 14:27:40,006] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_add_orgcom_name_abnormal_002
[2022-02-17 14:27:40,864] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_add_orgcom_name_abnormal_003
[2022-02-17 14:27:41,750] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_add_orgcom_code_abnormal_001
[2022-02-17 14:27:42,700] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_add_orgcom_code_abnormal_002
[2022-02-17 14:27:43,542] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_add_orgcom_code_abnormal_003
[2022-02-17 14:27:44,485] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_add_orgcom
[2022-02-17 14:27:46,356] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_edit_orgcom_name_abnormal_001
[2022-02-17 14:27:47,056] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_edit_orgcom_name_abnormal_002
[2022-02-17 14:27:49,127] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_edit_orgcom_name_abnormal_003
[2022-02-17 14:27:50,100] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_org_username_add_abnormal_001
[2022-02-17 14:27:53,547] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_org_username_add_abnormal_002
[2022-02-17 14:27:56,290] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_org_username_add_abnormal_003
[2022-02-17 14:27:59,056] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_org_code_add_adnormal_001
[2022-02-17 14:28:01,996] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_org_code_add_adnormal_002
[2022-02-17 14:28:04,938] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_org_code_add_adnormal_003
[2022-02-17 14:28:07,958] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_org_code_add_adnormal_004
[2022-02-17 14:28:10,966] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_org_code_add_adnormal_005
[2022-02-17 14:28:13,955] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_org_tel_add_abnormal_001
[2022-02-17 14:28:16,878] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_org_tel_add_abnormal_002
[2022-02-17 14:28:19,699] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_org_tel_add_abnormal_003
[2022-02-17 14:28:22,611] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_org_tel_add_abnormal_004
[2022-02-17 14:28:25,573] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_org_username_edit_abnormal_001
[2022-02-17 14:29:13,077] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_org_username_edit_abnormal_002
[2022-02-17 14:29:54,520] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_org_username_edit_abnormal_003
[2022-02-17 14:30:35,811] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_org_tel_edit_abmormal_001
[2022-02-17 14:31:17,210] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_org_tel_edit_abmormal_002
[2022-02-17 14:32:26,697] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_add_orgcom_name_abnormal_001
[2022-02-17 14:32:28,443] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_add_orgcom_name_abnormal_002
[2022-02-17 14:32:29,313] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_add_orgcom_name_abnormal_003
[2022-02-17 14:32:30,221] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_add_orgcom_code_abnormal_001
[2022-02-17 14:32:31,140] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_add_orgcom_code_abnormal_002
[2022-02-17 14:32:32,013] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_add_orgcom_code_abnormal_003
[2022-02-17 14:32:32,917] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_add_orgcom
[2022-02-17 14:32:34,773] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_edit_orgcom_name_abnormal_001
[2022-02-17 14:32:35,442] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_edit_orgcom_name_abnormal_002
[2022-02-17 14:32:36,446] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_edit_orgcom_name_abnormal_003
[2022-02-17 14:32:38,489] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_org_username_add_abnormal_001
[2022-02-17 14:32:41,609] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_org_username_add_abnormal_002
[2022-02-17 14:32:44,568] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_org_username_add_abnormal_003
[2022-02-17 14:32:47,413] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_org_code_add_adnormal_001
[2022-02-17 14:32:50,242] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_org_code_add_adnormal_002
[2022-02-17 14:32:53,195] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_org_code_add_adnormal_003
[2022-02-17 14:32:56,110] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_org_code_add_adnormal_004
[2022-02-17 14:32:59,063] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_org_code_add_adnormal_005
[2022-02-17 14:33:02,045] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_org_tel_add_abnormal_001
[2022-02-17 14:33:04,973] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_org_tel_add_abnormal_002
[2022-02-17 14:33:07,831] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_org_tel_add_abnormal_003
[2022-02-17 14:33:10,749] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_org_tel_add_abnormal_004
[2022-02-17 14:33:13,612] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_org_username_edit_abnormal_001
[2022-02-17 14:33:26,705] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_org_username_edit_abnormal_002
[2022-02-17 14:33:27,697] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_org_username_edit_abnormal_003
[2022-02-17 14:33:28,685] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_org_tel_edit_abmormal_001
[2022-02-17 14:33:29,627] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_org_tel_edit_abmormal_002
[2022-02-17 14:33:30,504] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_org_tel_edit_abmormal_003
[2022-02-17 14:33:31,505] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_org_tel_edit_abmormal_004
[2022-02-17 14:40:06,008] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_add_orgcom_name_abnormal_001
[2022-02-17 14:40:08,017] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_add_orgcom_name_abnormal_002
[2022-02-17 14:40:08,889] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_add_orgcom_name_abnormal_003
[2022-02-17 14:40:09,849] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_add_orgcom_code_abnormal_001
[2022-02-17 14:40:10,716] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_add_orgcom_code_abnormal_002
[2022-02-17 14:40:11,621] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_add_orgcom_code_abnormal_003
[2022-02-17 14:40:12,515] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_add_orgcom
[2022-02-17 14:40:14,373] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_edit_orgcom_name_abnormal_001
[2022-02-17 14:40:15,207] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_edit_orgcom_name_abnormal_002
[2022-02-17 14:40:16,205] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_edit_orgcom_name_abnormal_003
[2022-02-17 14:40:18,250] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_org_username_add_abnormal_001
[2022-02-17 14:40:21,390] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_org_username_add_abnormal_002
[2022-02-17 14:40:24,554] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_org_username_add_abnormal_003
[2022-02-17 14:40:27,604] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_org_code_add_adnormal_001
[2022-02-17 14:40:30,540] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_org_code_add_adnormal_002
[2022-02-17 14:40:33,389] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_org_code_add_adnormal_003
[2022-02-17 14:40:36,423] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_org_code_add_adnormal_004
[2022-02-17 14:40:39,554] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_org_code_add_adnormal_005
[2022-02-17 14:40:42,480] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_org_tel_add_abnormal_001
[2022-02-17 14:40:45,396] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_org_tel_add_abnormal_002
[2022-02-17 14:40:48,283] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_org_tel_add_abnormal_003
[2022-02-17 14:40:51,201] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_org_tel_add_abnormal_004
[2022-02-17 14:40:54,174] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_org_username_edit_abnormal_001
[2022-02-17 14:41:02,793] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_org_username_edit_abnormal_002
[2022-02-17 14:41:03,710] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_org_username_edit_abnormal_003
[2022-02-17 14:41:04,678] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_org_tel_edit_abmormal_001
[2022-02-17 14:41:05,625] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_org_tel_edit_abmormal_002
[2022-02-17 14:41:06,536] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_org_tel_edit_abmormal_003
[2022-02-17 14:41:07,553] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_org_tel_edit_abmormal_004
[2022-02-17 14:48:41,415] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_add_orgcom_name_abnormal_001
[2022-02-17 14:48:43,198] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_add_orgcom_name_abnormal_002
[2022-02-17 14:48:44,005] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_add_orgcom_name_abnormal_003
[2022-02-17 14:48:44,856] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_add_orgcom_code_abnormal_001
[2022-02-17 14:48:45,673] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_add_orgcom_code_abnormal_002
[2022-02-17 14:48:46,494] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_add_orgcom_code_abnormal_003
[2022-02-17 14:48:47,306] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_add_orgcom
[2022-02-17 14:48:49,190] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_edit_orgcom_name_abnormal_001
[2022-02-17 14:48:49,746] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_edit_orgcom_name_abnormal_002
[2022-02-17 14:48:50,606] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_edit_orgcom_name_abnormal_003
[2022-02-17 14:48:52,591] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_org_username_add_abnormal_001
[2022-02-17 14:48:55,750] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_org_username_add_abnormal_002
[2022-02-17 14:48:58,659] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_org_username_add_abnormal_003
[2022-02-17 14:49:01,739] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_org_code_add_adnormal_001
[2022-02-17 14:49:04,803] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_org_code_add_adnormal_002
[2022-02-17 14:49:07,699] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_org_code_add_adnormal_003
[2022-02-17 14:49:10,494] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_org_code_add_adnormal_004
[2022-02-17 14:49:13,328] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_org_code_add_adnormal_005
[2022-02-17 14:49:16,286] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_org_tel_add_abnormal_001
[2022-02-17 14:49:19,161] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_org_tel_add_abnormal_002
[2022-02-17 14:49:22,044] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_org_tel_add_abnormal_003
[2022-02-17 14:49:25,003] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_org_tel_add_abnormal_004
[2022-02-17 14:49:28,169] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_org_username_edit_abnormal_001
[2022-02-17 14:49:34,422] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_org_username_edit_abnormal_002
[2022-02-17 14:49:35,331] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_org_username_edit_abnormal_003
[2022-02-17 14:49:36,290] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_org_tel_edit_abmormal_001
[2022-02-17 14:49:37,161] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_org_tel_edit_abmormal_002
[2022-02-17 14:49:38,091] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_org_tel_edit_abmormal_003
[2022-02-17 14:49:39,088] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_org_tel_edit_abmormal_004
[2022-02-17 14:51:08,619] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_add_orgcom_name_abnormal_001
[2022-02-17 14:51:10,337] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_add_orgcom_name_abnormal_002
[2022-02-17 14:51:11,155] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_add_orgcom_name_abnormal_003
[2022-02-17 14:51:11,980] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_add_orgcom_code_abnormal_001
[2022-02-17 14:51:12,770] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_add_orgcom_code_abnormal_002
[2022-02-17 14:51:13,583] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_add_orgcom_code_abnormal_003
[2022-02-17 14:51:14,455] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_add_orgcom
[2022-02-17 14:51:16,371] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_edit_orgcom_name_abnormal_001
[2022-02-17 14:51:16,950] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_edit_orgcom_name_abnormal_002
[2022-02-17 14:51:17,806] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_edit_orgcom_name_abnormal_003
[2022-02-17 14:51:18,722] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_org_username_add_abnormal_001
[2022-02-17 14:51:21,867] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_org_username_add_abnormal_002
[2022-02-17 14:51:24,914] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_org_username_add_abnormal_003
[2022-02-17 14:51:27,854] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_org_code_add_adnormal_001
[2022-02-17 14:51:30,822] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_org_code_add_adnormal_002
[2022-02-17 14:51:33,731] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_org_code_add_adnormal_003
[2022-02-17 14:51:36,757] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_org_code_add_adnormal_004
[2022-02-17 14:51:39,806] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_org_code_add_adnormal_005
[2022-02-17 14:51:42,745] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_org_tel_add_abnormal_001
[2022-02-17 14:51:45,633] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_org_tel_add_abnormal_002
[2022-02-17 14:51:48,513] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_org_tel_add_abnormal_003
[2022-02-17 14:51:51,438] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_org_tel_add_abnormal_004
[2022-02-17 14:51:54,296] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_org_username_edit_abnormal_001
[2022-02-17 14:52:00,520] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_org_username_edit_abnormal_002
[2022-02-17 14:52:01,437] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_org_username_edit_abnormal_003
[2022-02-17 14:52:02,376] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_org_tel_edit_abmormal_001
[2022-02-17 14:52:03,270] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_org_tel_edit_abmormal_002
[2022-02-17 14:52:04,138] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_org_tel_edit_abmormal_003
[2022-02-17 14:52:05,079] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_org_tel_edit_abmormal_004
[2022-02-17 14:57:31,013] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_add_usergp
[2022-02-17 14:57:36,001] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_setauth_usergp
[2022-02-17 14:57:40,402] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_edit_usergp
[2022-02-17 14:57:44,551] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_delete_usergp
[2022-02-17 14:58:07,928] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_add_usergp
[2022-02-17 14:58:12,961] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_setauth_usergp
[2022-02-17 14:58:17,735] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_edit_usergp
[2022-02-17 14:58:22,336] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_delete_usergp
[2022-02-17 14:59:20,355] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_add_usergp
[2022-02-17 14:59:28,911] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_setauth_usergp
[2022-02-17 14:59:33,110] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_edit_usergp
[2022-02-17 14:59:36,884] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_delete_usergp
[2022-02-17 15:44:05,598] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_add_usergp
[2022-02-17 15:44:27,612] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_setauth_usergp
[2022-02-17 15:44:31,880] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_edit_usergp
[2022-02-17 15:44:35,747] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_delete_usergp
[2022-02-17 15:45:17,526] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_add_usergp
[2022-02-17 15:45:24,569] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_setauth_usergp
[2022-02-17 15:45:28,718] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_edit_usergp
[2022-02-17 15:45:32,579] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_delete_usergp
[2022-02-17 15:47:35,270] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_add_usergp
[2022-02-17 15:47:39,941] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_setauth_usergp
[2022-02-17 15:47:49,648] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_edit_usergp
[2022-02-17 15:47:56,062] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_delete_usergp
[2022-02-17 15:49:22,053] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_add_usergp
[2022-02-17 15:49:28,355] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_setauth_usergp
[2022-02-17 15:49:33,693] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_edit_usergp
[2022-02-17 15:49:37,625] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_delete_usergp
[2022-02-17 15:53:34,106] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_add_auth
[2022-02-17 15:57:42,325] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_add_auth
[2022-02-17 15:57:56,989] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_edit_auth
[2022-02-17 15:58:00,609] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_delete_auth
[2022-02-17 16:00:22,064] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_add_user
[2022-02-17 16:01:08,583] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_search_user
[2022-02-17 16:15:37,003] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_add_user
[2022-02-17 16:15:44,881] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_search_user
[2022-02-17 16:15:54,254] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_edit_user
[2022-02-17 16:16:17,286] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_active_user
[2022-02-17 16:16:21,125] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_lock_user
[2022-02-17 16:17:02,451] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_unlock_user
[2022-02-17 16:17:46,721] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_delete_user
[2022-02-17 16:32:04,056] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_add_user
[2022-02-17 16:32:10,874] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_search_user
[2022-02-17 16:32:20,198] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_edit_user
[2022-02-17 16:32:24,793] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_active_user
[2022-02-17 16:32:25,836] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_lock_user
[2022-02-17 16:33:16,754] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_unlock_user
[2022-02-17 16:33:58,174] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_delete_user
[2022-02-17 16:34:56,180] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_add_user
[2022-02-17 16:35:02,247] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_search_user
[2022-02-17 16:35:11,518] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_edit_user
[2022-02-17 16:35:16,353] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_active_user
[2022-02-17 16:35:55,582] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_lock_user
[2022-02-17 16:36:00,960] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_unlock_user
[2022-02-17 16:36:06,336] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_delete_user
[2022-02-17 16:41:14,079] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_adduser_usernamerror1
[2022-02-17 16:41:19,010] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_adduser_usernamerror1
[2022-02-17 16:41:23,721] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_adduser_usernamerror1
[2022-02-17 16:42:12,966] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_adduser_usernamerror1
[2022-02-17 16:42:19,436] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_adduser_usernamerror1
[2022-02-17 16:42:25,777] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_adduser_usernamerror1
[2022-02-17 16:42:32,105] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_adduser_usernamerror1
[2022-02-17 16:42:38,416] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_adduser_usernamerror1
[2022-02-17 16:42:44,671] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_adduser_usernamerror1
[2022-02-17 16:42:51,013] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_adduser_usernamerror1
[2022-02-17 16:42:57,409] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_adduser_usernamerror2
[2022-02-17 16:43:03,718] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_adduser_mailerror
[2022-02-17 16:43:09,940] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_adduser_mailerror
[2022-02-17 16:43:16,229] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_adduser_mailerror
[2022-02-17 16:43:22,461] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_adduser_mailerror
[2022-02-17 16:43:28,711] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_adduser_mailerror
[2022-02-17 16:43:34,928] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_adduser_rfiderror
[2022-02-17 16:43:41,002] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_edituser_mailerror
[2022-02-17 16:43:46,737] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_edituser_mailerror
[2022-02-17 16:43:52,463] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_edituser_mailerror
[2022-02-17 16:43:58,495] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_edituser_mailerror
[2022-02-17 16:44:04,450] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_edituser_mailerror
[2022-02-17 16:44:10,401] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_activeuser_passwderror1
[2022-02-17 16:44:11,762] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_activeuser_passwderror2
[2022-02-17 16:46:02,066] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_login_organization
[2022-02-17 16:46:06,819] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_add_orgcom
[2022-02-17 16:46:08,564] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_edit_orgcom
[2022-02-17 16:46:17,414] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_delete_orgcom
[2022-02-17 16:46:18,593] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_refresh_orgcom
[2022-02-17 16:46:18,665] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_org_user_edit
[2022-02-17 16:46:27,697] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_org_user_delete
[2022-02-17 16:47:09,451] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_org_refresh
[2022-02-17 16:48:17,280] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_login_organization
[2022-02-17 16:48:19,718] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_add_orgcom
[2022-02-17 16:48:21,788] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_edit_orgcom
[2022-02-17 16:48:23,367] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_delete_orgcom
[2022-02-17 16:48:24,568] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_refresh_orgcom
[2022-02-17 16:48:24,660] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_org_user_edit
[2022-02-17 16:48:31,870] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_org_user_delete
[2022-02-17 16:49:13,669] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_org_refresh
[2022-02-17 16:49:47,800] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_login_organization
[2022-02-17 16:49:52,524] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_add_orgcom
[2022-02-17 16:49:54,200] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_edit_orgcom
[2022-02-17 16:49:56,780] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_delete_orgcom
[2022-02-17 16:49:57,976] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_refresh_orgcom
[2022-02-17 16:49:58,047] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_org_user_edit
[2022-02-17 16:50:05,108] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_org_user_delete
[2022-02-17 16:50:11,437] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_org_refresh
[2022-02-17 16:50:57,705] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_login_organization
[2022-02-17 16:50:59,046] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_add_orgcom
[2022-02-17 16:51:00,697] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_edit_orgcom
[2022-02-17 16:51:02,302] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_delete_orgcom
[2022-02-17 16:51:03,505] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_refresh_orgcom
[2022-02-17 16:51:03,622] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_org_user_edit
[2022-02-17 16:51:13,035] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_org_user_delete
[2022-02-17 16:51:21,764] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_org_refresh
[2022-02-17 16:52:07,179] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_login_organization
[2022-02-17 16:52:10,436] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_add_orgcom
[2022-02-17 16:52:12,174] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_edit_orgcom
[2022-02-17 16:52:13,775] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_delete_orgcom
[2022-02-17 16:52:14,972] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_refresh_orgcom
[2022-02-17 16:52:15,095] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_org_user_edit
[2022-02-17 16:52:23,540] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_org_user_delete
[2022-02-17 16:52:23,638] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_org_refresh
[2022-02-17 16:54:46,593] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_add_orgcom_name_abnormal_001
[2022-02-17 16:54:48,344] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_add_orgcom_name_abnormal_002
[2022-02-17 16:54:49,092] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_add_orgcom_name_abnormal_003
[2022-02-17 16:54:49,966] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_add_orgcom_code_abnormal_001
[2022-02-17 16:54:50,795] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_add_orgcom_code_abnormal_002
[2022-02-17 16:54:51,656] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_add_orgcom_code_abnormal_003
[2022-02-17 16:54:52,500] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_add_orgcom
[2022-02-17 16:54:54,378] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_edit_orgcom_name_abnormal_001
[2022-02-17 16:54:54,970] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_edit_orgcom_name_abnormal_002
[2022-02-17 16:54:55,813] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_edit_orgcom_name_abnormal_003
[2022-02-17 16:54:56,687] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_org_username_add_abnormal_001
[2022-02-17 16:54:59,743] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_org_username_add_abnormal_002
[2022-02-17 16:55:02,679] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_org_username_add_abnormal_003
[2022-02-17 16:55:05,700] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_org_code_add_adnormal_001
[2022-02-17 16:55:08,552] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_org_code_add_adnormal_002
[2022-02-17 16:55:11,446] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_org_code_add_adnormal_003
[2022-02-17 16:55:14,394] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_org_code_add_adnormal_004
[2022-02-17 16:55:17,307] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_org_code_add_adnormal_005
[2022-02-17 16:55:20,237] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_org_tel_add_abnormal_001
[2022-02-17 16:55:23,149] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_org_tel_add_abnormal_002
[2022-02-17 16:55:25,940] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_org_tel_add_abnormal_003
[2022-02-17 16:55:28,741] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_org_tel_add_abnormal_004
[2022-02-17 16:55:31,654] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_org_username_edit_abnormal_001
[2022-02-17 16:55:42,571] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_org_username_edit_abnormal_002
[2022-02-17 16:55:45,548] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_org_username_edit_abnormal_003
[2022-02-17 16:55:48,486] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_org_tel_edit_abmormal_001
[2022-02-17 16:55:51,376] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_org_tel_edit_abmormal_002
[2022-02-17 16:55:54,274] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_org_tel_edit_abmormal_003
[2022-02-17 16:55:57,200] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_org_tel_edit_abmormal_004
[2022-02-17 16:59:23,346] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_add_orgcom_name_abnormal_001
[2022-02-17 16:59:25,218] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_add_orgcom_name_abnormal_001
[2022-02-17 16:59:26,016] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_add_orgcom_name_abnormal_001
[2022-02-17 17:10:17,803] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_add_orgcom_name_abnormal
[2022-02-17 17:10:20,508] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_add_orgcom_name_abnormal
[2022-02-17 17:10:21,369] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_add_orgcom_name_abnormal
[2022-02-17 17:10:22,214] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_add_orgcom_code_abnormal
[2022-02-17 17:10:23,029] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_add_orgcom_code_abnormal
[2022-02-17 17:10:23,855] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_add_orgcom_code_abnormal
[2022-02-17 17:10:24,725] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_add_orgcom
[2022-02-17 17:10:26,582] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_edit_orgcom_name_abnormal
[2022-02-17 17:10:27,198] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_edit_orgcom_name_abnormal
[2022-02-17 17:10:27,964] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_edit_orgcom_name_abnormal
[2022-02-17 17:10:28,786] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_org_username_add_abnormal
[2022-02-17 17:10:31,827] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_org_username_add_abnormal
[2022-02-17 17:10:34,743] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_org_username_add_abnormal
[2022-02-17 17:10:37,658] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_org_code_add_adnormal
[2022-02-17 17:10:40,505] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_org_code_add_adnormal
[2022-02-17 17:10:43,202] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_org_code_add_adnormal
[2022-02-17 17:10:47,749] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_org_code_add_adnormal
[2022-02-17 17:10:50,604] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_org_code_add_adnormal
[2022-02-17 17:10:53,568] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_org_tel_add_abnormal
[2022-02-17 17:10:56,439] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_org_tel_add_abnormal
[2022-02-17 17:10:59,293] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_org_tel_add_abnormal
[2022-02-17 17:11:02,197] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_org_tel_add_abnormal
[2022-02-17 17:11:05,129] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_org_username_edit_abnormal
[2022-02-17 17:12:14,030] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_org_username_edit_abnormal
[2022-02-17 17:13:09,572] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_org_username_edit_abnormal
[2022-02-17 17:14:01,542] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_org_tel_edit_abmormal
[2022-02-17 17:14:42,926] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_org_tel_edit_abmormal
[2022-02-17 17:14:58,161] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_org_tel_edit_abmormal
[2022-02-17 17:15:01,162] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_org_tel_edit_abmormal
[2022-02-17 17:15:26,620] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_add_orgcom_name_abnormal
[2022-02-17 17:15:28,366] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_add_orgcom_name_abnormal
[2022-02-17 17:15:29,186] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_add_orgcom_name_abnormal
[2022-02-17 17:15:30,035] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_add_orgcom_code_abnormal
[2022-02-17 17:15:30,858] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_add_orgcom_code_abnormal
[2022-02-17 17:15:31,685] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_add_orgcom_code_abnormal
[2022-02-17 17:15:32,483] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_add_orgcom
[2022-02-17 17:15:34,266] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_edit_orgcom_name_abnormal
[2022-02-17 17:15:34,852] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_edit_orgcom_name_abnormal
[2022-02-17 17:15:35,615] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_edit_orgcom_name_abnormal
[2022-02-17 17:15:36,441] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_org_username_add_abnormal
[2022-02-17 17:15:39,494] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_org_username_add_abnormal
[2022-02-17 17:15:42,438] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_org_username_add_abnormal
[2022-02-17 17:15:45,380] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_org_code_add_adnormal
[2022-02-17 17:15:48,075] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_org_code_add_adnormal
[2022-02-17 17:15:50,933] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_org_code_add_adnormal
[2022-02-17 17:15:53,846] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_org_code_add_adnormal
[2022-02-17 17:15:56,720] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_org_code_add_adnormal
[2022-02-17 17:15:59,664] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_org_tel_add_abnormal
[2022-02-17 17:16:02,486] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_org_tel_add_abnormal
[2022-02-17 17:16:05,269] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_org_tel_add_abnormal
[2022-02-17 17:16:08,219] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_org_tel_add_abnormal
[2022-02-17 17:16:11,102] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_add_user
[2022-02-17 17:16:34,235] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_org_tel_edit_abmormal
[2022-02-17 17:16:37,170] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_org_tel_edit_abmormal
[2022-02-17 17:16:40,128] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_org_tel_edit_abmormal
[2022-02-17 17:16:43,028] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_org_tel_edit_abmormal
[2022-02-17 17:19:07,890] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_add_orgcom_name_abnormal
[2022-02-17 17:19:10,613] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_add_orgcom_name_abnormal
[2022-02-17 17:19:11,549] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_add_orgcom_name_abnormal
[2022-02-17 17:19:12,395] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_add_orgcom_code_abnormal
[2022-02-17 17:19:13,182] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_add_orgcom_code_abnormal
[2022-02-17 17:19:14,031] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_add_orgcom_code_abnormal
[2022-02-17 17:19:14,879] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_add_orgcom
[2022-02-17 17:19:16,722] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_edit_orgcom_name_abnormal
[2022-02-17 17:19:17,287] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_edit_orgcom_name_abnormal
[2022-02-17 17:19:37,912] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_edit_orgcom_name_abnormal
[2022-02-17 17:19:38,458] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_org_username_add_abnormal
[2022-02-17 17:19:41,495] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_org_username_add_abnormal
[2022-02-17 17:19:44,387] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_org_username_add_abnormal
[2022-02-17 17:19:47,324] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_org_code_add_adnormal
[2022-02-17 17:19:50,223] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_org_code_add_adnormal
[2022-02-17 17:19:53,097] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_org_code_add_adnormal
[2022-02-17 17:19:55,979] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_org_code_add_adnormal
[2022-02-17 17:19:58,890] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_org_code_add_adnormal
[2022-02-17 17:20:01,753] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_org_tel_add_abnormal
[2022-02-17 17:20:04,673] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_org_tel_add_abnormal
[2022-02-17 17:20:07,488] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_org_tel_add_abnormal
[2022-02-17 17:20:10,360] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_org_tel_add_abnormal
[2022-02-17 17:20:13,233] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_add_user
[2022-02-17 17:20:33,816] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_org_tel_edit_abmormal
[2022-02-17 17:20:36,766] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_org_tel_edit_abmormal
[2022-02-17 17:20:39,663] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_org_tel_edit_abmormal
[2022-02-17 17:20:42,718] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_org_tel_edit_abmormal
[2022-02-17 17:26:10,009] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_add_auth
[2022-02-17 17:26:22,830] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_edit_auth
[2022-02-17 17:26:26,421] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_delete_auth
[2022-02-17 17:26:28,623] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_login_organization
[2022-02-17 17:26:29,700] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_add_orgcom
[2022-02-17 17:26:33,722] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_edit_orgcom
[2022-02-17 17:27:15,244] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_delete_orgcom
[2022-02-17 17:27:35,567] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_refresh_orgcom
[2022-02-17 17:28:17,159] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_org_user_edit
[2022-02-17 17:28:59,726] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_org_user_delete
[2022-02-17 17:29:41,314] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_org_refresh
[2022-02-17 17:31:07,959] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_add_orgcom_name_abnormal
[2022-02-17 17:31:49,910] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_add_orgcom_name_abnormal
[2022-02-17 17:32:31,759] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_add_orgcom_name_abnormal
[2022-02-17 17:33:13,707] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_add_orgcom_code_abnormal
[2022-02-17 17:34:14,718] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_add_auth
[2022-02-17 17:34:27,710] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_edit_auth
[2022-02-17 17:34:31,363] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_delete_auth
[2022-02-17 17:34:33,605] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_login_organization
[2022-02-17 17:34:34,691] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_add_orgcom
[2022-02-17 17:34:39,168] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_edit_orgcom
[2022-02-17 17:35:06,153] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_delete_orgcom
[2022-02-17 17:35:42,810] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_add_auth
[2022-02-17 17:35:55,520] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_edit_auth
[2022-02-17 17:35:59,322] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_delete_auth
[2022-02-17 17:36:01,576] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_login_organization
[2022-02-17 17:36:02,664] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_add_orgcom
[2022-02-17 17:36:07,084] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_edit_orgcom
[2022-02-17 17:36:11,097] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_delete_orgcom
[2022-02-17 17:37:40,082] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_login_organization
[2022-02-17 17:37:42,188] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_add_orgcom
[2022-02-17 17:37:45,936] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_edit_orgcom
[2022-02-17 17:37:49,690] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_delete_orgcom
[2022-02-17 17:37:50,897] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_refresh_orgcom
[2022-02-17 17:37:50,996] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_org_user_edit
[2022-02-17 17:37:59,346] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_org_user_delete
[2022-02-17 17:37:59,506] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_org_refresh
[2022-02-17 17:38:22,558] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_add_orgcom_name_abnormal
[2022-02-17 17:38:26,987] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_add_orgcom_name_abnormal
[2022-02-17 17:38:27,945] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_add_orgcom_name_abnormal
[2022-02-17 17:38:28,724] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_add_orgcom_code_abnormal
[2022-02-17 17:38:29,565] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_add_orgcom_code_abnormal
[2022-02-17 17:38:30,419] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_add_orgcom_code_abnormal
[2022-02-17 17:38:31,298] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_add_orgcom
[2022-02-17 17:38:33,151] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_edit_orgcom_name_abnormal
[2022-02-17 17:38:33,713] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_edit_orgcom_name_abnormal
[2022-02-17 17:38:46,871] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_edit_orgcom_name_abnormal
[2022-02-17 17:38:50,881] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_org_username_add_abnormal
[2022-02-17 17:38:54,020] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_org_username_add_abnormal
[2022-02-17 17:38:56,933] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_org_username_add_abnormal
[2022-02-17 17:38:59,986] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_org_code_add_adnormal
[2022-02-17 17:39:02,893] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_org_code_add_adnormal
[2022-02-17 17:39:05,869] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_org_code_add_adnormal
[2022-02-17 17:39:08,802] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_org_code_add_adnormal
[2022-02-17 17:39:11,675] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_org_code_add_adnormal
[2022-02-17 17:39:14,420] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_org_tel_add_abnormal
[2022-02-17 17:39:17,052] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_org_tel_add_abnormal
[2022-02-17 17:39:19,881] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_org_tel_add_abnormal
[2022-02-17 17:39:22,791] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_org_tel_add_abnormal
[2022-02-17 17:39:25,801] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_add_user
[2022-02-17 17:39:46,072] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_org_tel_edit_abmormal
[2022-02-17 17:39:49,011] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_org_tel_edit_abmormal
[2022-02-17 17:39:51,967] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_org_tel_edit_abmormal
[2022-02-17 17:39:54,899] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_org_tel_edit_abmormal
[2022-02-17 17:40:39,401] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_add_orgcom_name_abnormal
[2022-02-17 17:40:42,197] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_add_orgcom_name_abnormal
[2022-02-17 17:40:43,057] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_add_orgcom_name_abnormal
[2022-02-17 17:40:43,909] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_add_orgcom_code_abnormal
[2022-02-17 17:40:44,713] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_add_orgcom_code_abnormal
[2022-02-17 17:40:45,549] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_add_orgcom_code_abnormal
[2022-02-17 17:40:46,471] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_add_orgcom
[2022-02-17 17:40:48,324] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_edit_orgcom_name_abnormal
[2022-02-17 17:41:08,572] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_edit_orgcom_name_abnormal
[2022-02-17 17:41:18,380] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_edit_orgcom_name_abnormal
[2022-02-17 17:41:19,143] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_org_username_add_abnormal
[2022-02-17 17:41:22,196] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_org_username_add_abnormal
[2022-02-17 17:41:25,098] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_org_username_add_abnormal
[2022-02-17 17:41:28,007] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_org_code_add_adnormal
[2022-02-17 17:41:30,855] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_org_code_add_adnormal
[2022-02-17 17:41:33,816] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_org_code_add_adnormal
[2022-02-17 17:41:36,882] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_org_code_add_adnormal
[2022-02-17 17:41:39,718] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_org_code_add_adnormal
[2022-02-17 17:41:42,580] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_org_tel_add_abnormal
[2022-02-17 17:41:45,324] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_org_tel_add_abnormal
[2022-02-17 17:41:48,061] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_org_tel_add_abnormal
[2022-02-17 17:41:51,021] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_org_tel_add_abnormal
[2022-02-17 17:41:54,014] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_add_user
[2022-02-17 17:42:14,170] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_org_tel_edit_abmormal
[2022-02-17 17:42:17,103] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_org_tel_edit_abmormal
[2022-02-17 17:42:20,001] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_org_tel_edit_abmormal
[2022-02-17 17:42:22,995] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_org_tel_edit_abmormal
[2022-02-17 17:42:46,291] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_add_orgcom_name_abnormal
[2022-02-17 17:42:47,933] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_add_orgcom_name_abnormal
[2022-02-17 17:42:48,734] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_add_orgcom_name_abnormal
[2022-02-17 17:42:49,571] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_add_orgcom_code_abnormal
[2022-02-17 17:42:50,369] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_add_orgcom_code_abnormal
[2022-02-17 17:42:51,160] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_add_orgcom_code_abnormal
[2022-02-17 17:42:51,991] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_add_orgcom
[2022-02-17 17:42:53,885] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_edit_orgcom_name_abnormal
[2022-02-17 17:42:54,446] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_edit_orgcom_name_abnormal
[2022-02-17 17:42:55,179] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_edit_orgcom_name_abnormal
[2022-02-17 17:42:55,917] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_org_username_add_abnormal
[2022-02-17 17:42:59,016] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_org_username_add_abnormal
[2022-02-17 17:43:02,007] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_org_username_add_abnormal
[2022-02-17 17:43:04,832] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_org_code_add_adnormal
[2022-02-17 17:43:07,597] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_org_code_add_adnormal
[2022-02-17 17:43:10,414] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_org_code_add_adnormal
[2022-02-17 17:43:13,536] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_org_code_add_adnormal
[2022-02-17 17:43:16,468] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_org_code_add_adnormal
[2022-02-17 17:43:19,326] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_org_tel_add_abnormal
[2022-02-17 17:43:22,170] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_org_tel_add_abnormal
[2022-02-17 17:43:25,069] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_org_tel_add_abnormal
[2022-02-17 17:43:28,043] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_org_tel_add_abnormal
[2022-02-17 17:43:30,902] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_add_user
[2022-02-17 17:43:51,171] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_org_tel_edit_abmormal
[2022-02-17 17:43:54,119] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_org_tel_edit_abmormal
[2022-02-17 17:43:57,122] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_org_tel_edit_abmormal
[2022-02-17 17:44:00,128] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_org_tel_edit_abmormal
[2022-02-17 17:44:24,847] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_login_organization
[2022-02-17 17:44:26,471] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_add_orgcom
[2022-02-17 17:44:28,040] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_edit_orgcom
[2022-02-17 17:44:34,801] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_delete_orgcom
[2022-02-17 17:44:36,005] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_refresh_orgcom
[2022-02-17 17:44:36,091] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_org_user_edit
[2022-02-17 17:44:44,584] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_org_user_delete
[2022-02-17 17:44:44,713] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_org_refresh
[2022-02-17 17:45:13,397] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_login_organization
[2022-02-17 17:45:18,201] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_add_orgcom
[2022-02-17 17:45:19,864] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_edit_orgcom
[2022-02-17 17:45:21,490] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_delete_orgcom
[2022-02-17 17:45:22,670] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_refresh_orgcom
[2022-02-17 17:45:22,740] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_org_user_edit
[2022-02-17 17:45:31,199] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_org_user_delete
[2022-02-17 17:45:31,304] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_org_refresh
[2022-02-17 17:46:08,403] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_add_auth
[2022-02-17 17:46:35,959] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_add_auth
[2022-02-17 17:46:49,205] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_edit_auth
[2022-02-17 17:46:52,770] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_delete_auth
[2022-02-17 17:46:54,999] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_login_organization
[2022-02-17 17:46:56,099] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_add_orgcom
[2022-02-17 17:47:00,553] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_edit_orgcom
[2022-02-17 17:47:04,355] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_delete_orgcom
[2022-02-17 17:47:08,154] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_refresh_orgcom
[2022-02-17 17:47:49,746] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_org_user_edit
[2022-02-17 17:48:32,505] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_org_user_delete
[2022-02-17 17:49:14,968] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_org_refresh
[2022-02-17 17:50:41,684] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_add_orgcom_name_abnormal
[2022-02-17 17:51:12,770] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_add_auth
[2022-02-17 17:51:26,932] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_edit_auth
[2022-02-17 17:51:30,579] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_delete_auth
[2022-02-17 17:51:32,823] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_login_organization
[2022-02-17 17:51:33,918] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_add_orgcom
[2022-02-17 17:51:35,579] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_edit_orgcom
[2022-02-17 17:51:37,178] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_delete_orgcom
[2022-02-17 17:51:38,407] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_refresh_orgcom
[2022-02-17 17:51:38,457] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_org_user_edit
[2022-02-17 17:51:43,444] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_org_user_delete
[2022-02-17 17:51:57,885] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_org_refresh
[2022-02-17 17:52:01,047] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_add_orgcom_name_abnormal
[2022-02-17 17:52:05,530] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_add_orgcom_name_abnormal
[2022-02-17 17:52:46,887] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_add_orgcom_name_abnormal
[2022-02-17 17:53:29,110] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_add_orgcom_code_abnormal
[2022-02-17 17:54:10,880] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_add_orgcom_code_abnormal
[2022-02-17 17:54:49,470] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_add_orgcom_code_abnormal
[2022-02-17 17:54:53,853] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_add_orgcom
[2022-02-17 17:54:55,676] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_edit_orgcom_name_abnormal
[2022-02-17 17:55:04,225] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_edit_orgcom_name_abnormal
[2022-02-17 17:55:10,855] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_edit_orgcom_name_abnormal
[2022-02-17 17:55:15,166] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_org_username_add_abnormal
[2022-02-17 17:55:28,379] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_org_username_add_abnormal
[2022-02-17 17:55:34,676] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_org_username_add_abnormal
[2022-02-17 17:56:16,714] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_org_code_add_adnormal
[2022-02-17 17:56:58,985] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_org_code_add_adnormal
[2022-02-17 17:57:41,011] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_org_code_add_adnormal
[2022-02-17 17:58:22,959] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_org_code_add_adnormal
[2022-02-17 17:59:04,956] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_org_code_add_adnormal
[2022-02-17 17:59:46,879] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_org_tel_add_abnormal
[2022-02-17 18:00:28,795] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_org_tel_add_abnormal
[2022-02-17 18:01:10,559] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_org_tel_add_abnormal
[2022-02-17 18:01:52,348] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_org_tel_add_abnormal
[2022-02-17 18:02:34,509] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_add_user
[2022-02-17 18:04:48,669] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_login_organization
[2022-02-17 18:04:52,166] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_add_orgcom
[2022-02-17 18:04:54,893] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_edit_orgcom
[2022-02-17 18:05:02,858] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_delete_orgcom
[2022-02-17 18:05:04,062] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_refresh_orgcom
[2022-02-17 18:05:04,133] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_org_user_edit
[2022-02-17 18:05:06,424] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_org_user_delete
[2022-02-17 18:06:00,766] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_login_organization
[2022-02-17 18:06:03,978] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_add_orgcom
[2022-02-17 18:06:05,671] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_edit_orgcom
[2022-02-17 18:06:07,277] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_delete_orgcom
[2022-02-17 18:06:08,471] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_refresh_orgcom
[2022-02-17 18:06:08,586] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_org_user_edit
[2022-02-17 18:06:14,191] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_org_user_delete
[2022-02-17 18:06:25,391] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_org_refresh
